LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2023 Kenny Tsui
+Copyright (c) 2025 Klaviyo
```

Comment:
Autogenerated by the template, no offense Kenny <3

Reply:
lolll